homeassistant/components/recorder/statistics.py

```diff
@@ -365,8 +366,10 @@ def delete_duplicates(instance: Recorder, session: scoped_session) -> None:
 
     if non_identical_duplicates:
         isotime = dt_util.utcnow().isoformat()
-        backup_file_name = f"deleted_statistics.{isotime}.json"
+        backup_file_name = f".deleted_statistics/deleted_statistics.{isotime}.json"
```

Comment:
How are people going to find it now?

Reply:
You get a warning logged with the full path to the location of the file it generated.

Reply:
People only check their logs when something is not working, not every time they start HA.

Reply:
And when they find that file, what then?